packages/account/src/Containers/Account/account.tsx

```diff
@@ -31,16 +31,30 @@ const Account = observer(({ history, location, routes }: TAccountProps) => {
         should_allow_authentication,
         should_allow_poinc_authentication,
     } = client;
-    const { toggleAccountSettings, is_account_settings_visible } = ui;
+    const { toggleAccountSettings, is_account_settings_visible, is_mobile, is_desktop } = ui;
+
+    const [available_routes, setAvailableRoutes] = React.useState(routes);
```

Comment:
Does this work @yauheni-deriv?
```suggestion
    const [available_routes, setAvailableRoutes] = React.useState<TRoute[]>(routes);
```

Reply:
not really @jim-deriv, because are different types for TRoute in account and in shared.
hopefully we will avoid such overlapping in v2